hash/ibm5170.xml

```diff
@@ -22197,6 +22350,47 @@ Disk 2 includes: "The Lords of Midnight (ZX Spectrum)", "Doomdark's Revenge (ZX
 		</part>
 	</software>
 
+	<!-- PC Today (UK) -->
+
+	<software name="pctod9212">
+		<description>PC Today (Vol. 06 № 08 - December 1992)</description>
+		<year>1992</year>
+		<publisher>EuroPress Publications</publisher>
+		<notes><![CDATA[
+Disk 1 includes: "GST TimeWorks Publisher Lite v1.01"
+Disk 2 includes: "Cliparts for TimeWorks Publisher Lite", "Wessex Systems Mail-Pro 2.01", "PC Today Buyer's Guide", "SysEdit"
+]]></notes>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="PC Today (Vol.06 N.08 - December 1992) [EuroPress] [1994] [3.5DD] [Disk 1 of 2].img" size="737280" crc="8649d92b" sha1="17151dbf03fe0765033f05e758c5dddfabf705f4"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="PC Today (Vol.06 N.08 - December 1992) [EuroPress] [1994] [3.5DD] [Disk 2 of 2].img" size="737280" crc="916e26c3" sha1="62492689cc89097bf2d437d70c72065d9dde10c4"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- Personal Computer World -->
+
+	<software name="pcw9404">
+		<description>Personal Computer World (1994-04 - April 1994)</description>
+		<year>1994</year>
+		<publisher>VNU Business Publications</publisher>
+		<notes><![CDATA[Cover Disk includes: 
+Amour-Geddon (part 2) [Psynosis]: Part 2 of this blockbuster game adds brilliant animation to the program's startup sequence.
+Marketeer [Sterling Data]: All the features anyone involved in marketing could ask for. A superb, easey-to-use system.
+Walking Fingers [David Geffen]: Let your fingers do the walking in this handy mini-database. Fun, logical and good graphics.
```

Comment:
"easey" should be "easy" (unless this is an intentional recreation of a misspelling).

Reply:
Also "Psynosis" should be "Psygnosis", if it isn't intentional.

Reply:
Thanks for the warnings. Corrections made.